magellan-library/src/main/java/com/wealthfront/magellan/view/DialogComponent.kt

```diff
@@ -29,8 +29,11 @@ public class DialogComponent @Inject constructor() : LifecycleAware {
     createDialog()
   }
 
-  override fun resume(context: Context) {
+  override fun show(context: Context) {
```

Comment:
The meat of the change: move context assignment to `show` from `resume`, and context release to `hide` from `pause`.